source/index.md

```diff
@@ -27,33 +27,33 @@ To get started, you can:
 
-* First, identify some legislation that can be expressed as an arithmetic operation.
-* Then, translate them into [formulas, variables, parameters, etc.](coding-the-legislation/index.md)
+* First, identify some legislation that includes arithmetic operations.
+* Then, translate it into [formulas, variables, parameters, etc.](coding-the-legislation/index.md)
 * [Build some tests](coding-the-legislation/writing_yaml_tests.md) to verify your implementation of the law.
 * Lost? You are not the first one to go through that! You can [find help](find-help.md) by reaching out to the community.
 
 ### 2 - Identify the input data you need
 
 With OpenFisca, you can calculate the effect of legislation on a single situation or a whole population by running a [simulation](simulate/index.md). Since the data you need depends on what you are trying to calculate, OpenFisca does not offer any data upfront.
 
-The following could be your use case to use OpenFisca:
+The following is an example OpenFisca use case:
 
 Do you want to help users find their eligibility for a social benefit in your country? Then, use OpenFisca to build a user interface asking them for their income and demographic information and provide them with an answer! (Do not forget to comply with GDPR!).
 
-Are you trying to assess the impact of a new housing tax on behalf of the OECD? Find your government's open survey data and use it with OpenFisca to simulate the effect of that tax reform on the poorest 20% of a country.
+Are you trying to assess the impact of a new housing tax on behalf of the OECD? Find your government's open survey data and use it with OpenFisca to simulate the effect of that tax reform on the poorest 20% of your country.
 
 ### 3 - Run Simulations
 
 With OpenFisca there are two ways to calculate the effect of the rules modelled in a country package on a given input data:
 
 * If you have a background in web development or want to build a web application with the results of your simulation, you’ll want to use the [web API](openfisca-web-api/index.md).
-* If you have a background in datascience, want to use large datasets, or want to dynamically apply changes to the system, you’ll rather use the [Python API](openfisca-python-api/index.md).
+* If you have a background in data science, want to use large datasets, or want to dynamically apply changes to the system, you should use the [Python API](openfisca-python-api/index.md).
 
 The output of this simulation will be either Python objects or JSON. To represent these results in a graphical format, you can make use of Python libraries such as [plot.ly](https://plot.ly).
 
-Please make sure you read our [license](license.md) before publishing results based on OpenFisca.
+Please ensure you read our [licence](license.md) before publishing results based on OpenFisca.
```

Comment:
Isn't `licence` American English?

Reply:
My understanding is the Americans have license. Outside of America licence is a noun, license is a verb.